src/components/Modal.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+
+export function Modal({isOpen, onClose, content}) {
+  const playerStyle = {
+    padding: '56.25% 0 0 0',
+    position: 'relative',
+  };
+
+  const iframeStyler = {
+    top: '0',
+    left: '0',
+    width: '100%',
+    height: '100%',
+    position: 'absolute',
+  };
```

Comment:
Since these styles don't change, you could define them outside of your `Modal` function call:
```suggestion
  const playerStyle = {
    padding: '56.25% 0 0 0',
    position: 'relative',
  };

  const iframeStyler = {
    top: '0',
    left: '0',
    width: '100%',
    height: '100%',
    position: 'absolute',
  };

export function Modal({isOpen, onClose, content}) {

```
On another thought: why don't you style them with styled-components? 🤔